app/apps/server/bridges/analytics.ts

```diff
@@ -0,0 +1,25 @@
+import { IAnalyticsPayload } from '@rocket.chat/apps-engine/definition/accessors';
+import { AnalyticsBridge } from '@rocket.chat/apps-engine/server/bridges/AnalyticsBridge';
+
+import { SNS } from '../../../livechat/server';
+import { AppServerOrchestrator } from '../orchestrator';
+
+export class AppAnalyticsBridge extends AnalyticsBridge {
+	constructor(private readonly orch: AppServerOrchestrator) {
+		super();
+	}
+
+	protected async sendEvent(data: IAnalyticsPayload, appId: string): Promise<boolean> {
+		this.orch.debugLog(`The App ${appId} is sending an event for Analytics`);
+
+		const app = this.orch.getManager()?.getOneById(appId);
+
+		if (!app) {
+			throw new Error('Invalid app provided');
+		}
+
+		SNS.sendEvent(data || {});
```

Comment:
Do we need to log empty events as well?

Reply:
Actually, I added it just in case the data is null. But it's typescript so it won't allow that. So we can remove it. Thanks 😄